migtests/tests/pg/multiple-schemas/validateAfterChanges

```diff
@@ -0,0 +1,138 @@
+	assert res_aggregate_inc_sum == 120
+
+	fetched_extension = tgt.execute_query("SELECT extname FROM pg_extension where extname='pgcrypto';")
+	print(f"fetched extension for extname - {fetched_extension}")
+	assert fetched_extension == 'pgcrypto'
+
+	chk_executed_query = tgt.run_query_and_chk_error("call tt_insert_data(6);", None)
+	print(f"error in executing query - {chk_executed_query}")
+	assert chk_executed_query == False
+
+	#after calling tt_insert_data() functions inserts happen on tt and audit table
+	cnt_tt = tgt.get_row_count("tt", schema_name)
+	print(f"after running the procedure to insert into tt table, row count - {cnt_tt}")
+	assert cnt_tt == EXPECTED_ROW_COUNT["tt"]+1
+
+	cnt_audit = tgt.get_row_count("audit", schema_name)
+	print(f"after running the procedure to insert into audit table, row count of audit- {cnt_audit}")
+	assert cnt_audit == EXPECTED_ROW_COUNT["audit"]+1
+
+	#checking the total again with changed value
+	res_total_proc = tgt.execute_query("select total();")
+	print(f"result of total() - {res_total_proc}")
+	assert res_total_proc == EXPECTED_TOTAL+6
+
+
+
+def migration_completed_checks(tgt):
+
+	check_validations_per_schema(tgt, "public")
+
+	QUERY_CHK_TYPES_DOMAINS = "select count(typname) from pg_type where typname in ('enum_kind', 'item_details', 'person_name');"
+	cnt_type_domain = tgt.execute_query(QUERY_CHK_TYPES_DOMAINS)
+	print(f"count returned - {cnt_type_domain}")
+	assert cnt_type_domain == 6
+
+	SET_SCHEMA_SEARCH_PATH_QUERY = "set search_path to schema2;"
+	chk_error_execute_set_schema = tgt.run_query_and_chk_error(SET_SCHEMA_SEARCH_PATH_QUERY, None)
+	print(f"error in executing the set searchpath query - {chk_error_execute_set_schema}")
+	assert chk_error_execute_set_schema == False
+
+	schema_set = tgt.execute_query("show search_path;")
+	print(f"schema set in search_path - {schema_set}")
+	assert schema_set == 'schema2'
+
+	global EXPECTED_ROW_COUNT
+	global EXPECTED_TOTAL
+	EXPECTED_ROW_COUNT = EXPECTED_ROW_COUNT_SCHEMA2
+	EXPECTED_TOTAL=EXPECTED_TOTAL_SCHEMA2
+	check_validations_per_schema(tgt, "schema2")
+
+	INSERT_QUERY_EXT_TEST = "insert into ext_test(password) values (crypt('tomspassword', gen_salt('bf')));"
+	chk_insert_error_ext_test = tgt.run_query_and_chk_error(INSERT_QUERY_EXT_TEST, None)
+	print(f"error in inserting in ext_test - {chk_insert_error_ext_test}")
+	assert chk_insert_error_ext_test == False
+
+	cnt_ext_test = tgt.get_row_count("ext_test", "schema2")
+	print(f"after running the procedure to insert into ext_test table, row count- {cnt_ext_test}")
+	assert cnt_ext_test == 2
+
+	QUERY_TO_CHECK_PASSWORD_OF_ID = "select id from ext_test where password = crypt('johnspassword', password);"
+	id_returned_ext_test = tgt.execute_query(QUERY_TO_CHECK_PASSWORD_OF_ID)
+	print(f"id returned for password(johnspassword) - {id_returned_ext_test}")
+	assert id_returned_ext_test == 1
+	
+	print("Validating the yb supported extensions list:")
```

Comment:
You can get rid of this validation from here it is fine to keep this just in `validate` only.